lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -1,10 +1,36 @@
 class ActiveModel::Serializer::Adapter::JsonApi < ActiveModel::Serializer::Adapter
+        # JsonApiObject: namespace for functions that build
+        # JSON API objects
+        module JsonApiObject
```

Comment:
This could be extracted to `adapter/json_api/json_api_object.rb`, what do you think?